enterprise/api_client/enterprise_catalog.py

```diff
@@ -312,6 +315,18 @@ def enterprise_contains_content_items(self, enterprise_uuid, content_ids):
         response.raise_for_status()
         return response.json()['contains_content_items']
 
+    @UserAPIClient.refresh_token
+    def get_content_metadata_content_identifier(self, enterprise_uuid, content_id):
+        """
+        Return all content metadata contained in the catalogs associated with the the
+        given EnterpriseCustomer and content_id.
+        """
+        api_url = self.get_api_url(
+            f"{self.CONTENT_METADATA_IDENTIFIER_ENDPOINT.format(enterprise_uuid, content_id)}")
+        response = self.client.get(api_url)
+        response.raise_for_status()
+        return response.json()
+
```

Comment:
Shouldn't we handle any exceptions here? like connection failure with enterprise-catalogs service. I'd suggest we take a look at this endpoint in enterprise-catalogs to see what kind of error does it raise.